tuneme/templates/new/patterns/basics/surveys/molo_survey_page.html

```diff
@@ -70,7 +70,7 @@ <h1 class="heading">{{ block.value }}</h1>
         </form>
     {% else %}
         <p class="error error--{{surveytype}}">
-          {% trans "You have already completed this" %} {{ surveytype|slice:":-1" }}
+          {% trans "You have already completed this" %} {% if surveytype|default:""|slice:"-1:" == "s" %}{{ surveytype|slice:":-1" }}{% else %}{{ surveytype }}{% endif %}
```

Comment:
Since we're using dynamic content here, administrators can use whatever close related word. We should probably not use the slicing of last word character.

Reply:
The admin can only specify what type of survey, the preset values are then _surveys_, _polls_ and _competition_, but yeah also found it a bit strange that the initial code had the last letter slice, also wont work with translations, I just made sure it works on words containing a last letter S... I've now updated it now to use its own separate value